packages/functions/src/donor/DonorBookAppointmentHandler.test.ts

```diff
@@ -57,6 +57,7 @@ test("Unauthenticated user throws exception", async () => {
 });
 
 test("Donor not found throws exception", async () => {
+  await saveAppointment(APPOINTMENT_TO_BOOK_2, false, 3);
```

Comment:
Why is this needed?

Reply:
Because the function checks if the appointment is available before it checks the donor. And if will not set an appointment the function will return NO_SUCH_APPOINTMENTS.